.github/workflows/main.yml

```
@@ -111,7 +114,35 @@ jobs:
 
       - name: Packaging
         run: |
-          cp .buildkite_win/appxmanifest.xml kolibri-windows/
+          # When running workflows for a tag, GitHub actions/checkout@v2 doesn't
+          # just check out the tag. Instead it does the following:
+          #  - pulls the repo
+          #  - uses ref-parse to determine the commit object the tag points at
+          #  - re-fetches with 'git fetch ... origin +$COMMIT_ID:refs/tags/$TAG'
+          #  - checks out that tag
+          # As a result it loses the signed tag, which means 'git describe'
+          # reports the version as the previous tag, plus a bunch of commits.
+          # Really weird.
+          #
+          # So we can't use 'git describe' and expect to get a bare tag back
+          # when we're building that tag. Happily, we can just get the tag name
+          # from the GitHub Actions environment and use that directly.
+          if (($env:GITHUB_ACTIONS -eq "true") -and ($env:GITHUB_REF_TYPE -eq "tag")) {
+              $describe = $env:GITHUB_REF_NAME
+          } else {
+              $describe = (git describe --always)
+          }
```

I just reread `git describe` and I wonder whether an alternative would be:

```suggestion
          $describe = (git describe --always --tags --match 'v*')
```

---

Hmm ... 
This makes things simpler!